invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/communities_items/ComputerTabletCommunitiesItem.js

```diff
@@ -18,7 +18,7 @@ export const ComputerTabletCommunitiesItem = ({ result, index }) => {
   const visibilityText = isPublic ? i18next.t("Public") : i18next.t("Restricted");
   const visibilityIcon = isPublic ? undefined : "ban";
   return (
-    <Item key={index} className="computer tablet only flex">
```

Comment:
This does not seem to be used by `app-rdm`. There is an implementation there and that's the one being used.
However, I fixed it in both places.